sample_code/ep544/rev05/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM bitnami/git
```

Comment:
Using the `bitnami/git` image because the `gbp` image in the video doesn't have git 🤷‍♂️ ...